resources/js/components/Modules/ModuleListing.vue

```diff
@@ -202,7 +213,8 @@ export default {
                 }
             ],
             showAllModules: !this.userLoggedIn,
-            currentBonusModules: this.filterBonusModules()
```

Comment:
Any particular reason why this was changed to a computed property? The `toggleShowAllModules` method updates the value of this property which will no longer work as a computed property.